zone/gm_commands/lootsim.cpp

```diff
@@ -0,0 +1,121 @@
+#include "../client.h"
+
+void command_lootsim(Client *c, const Seperator *sep)
+{
+	int arguments = sep->argnum;
+	if (arguments < 3) {
```

Comment:
Since we're not validating below, we can just stop them from getting there with invalid inputs. Or add validation.
`if (arguments <3 || !sep->IsNumber(1) || !sep->IsNumber(2) || !sep->IsNumber(3)) {`